moped-editor/src/views/projects/projectView/ProjectComponents/utils/form.js

```diff
@@ -248,6 +248,13 @@ export const useInitialValuesOnAttributesEdit = (
 
     setValue("tags", selectedTags);
   }, [componentTagsOptions, initialFormValues, setValue]);
+
+  // Set the srts id value
+  useEffect(() => {
+    if (!initialFormValues) return;
+
+    setValue("srtsId", initialFormValues.srtsId);
+  }, [initialFormValues, setValue]);
```

Comment:
@johnclary I think it might be the time to refactor how I wrote the existing value population in the form. Not sure how big of priority that you see this as, but I'd love to patch this up and squash that brief flash of old values that shows on save at the same time. 🚀

Reply:
agreed! would you hazard an estiamte? i'll queue it up. 
- https://github.com/cityofaustin/atd-data-tech/issues/11612

Reply:
great - thanks! i put a 5 on it since it might take some thinking and also touches a lot of the functionality in the component map.

Reply:
😈 🍄 🚀